pkg/obfuscate/sql_test.go

```diff
@@ -1159,6 +1159,103 @@ LIMIT 1
 	}
 }
 
+func TestPGJsonOperators(t *testing.T) {
+	assert := assert.New(t)
+	for _, tt := range []struct {
+		in, out string
+		cfg     SQLConfig
```

Comment:
What is the purpose of config if all the tests set it to `DBMSPostgreSQL`? Perhaps you should add more tests which don't set it at all, to clarify that there are no regressions.

Reply:
There were a couple tests with DBMSSQLServer, but I've added some more with no DBMS set.

Reply:
I'd propose the following, it will result in a better structure:
* Keep all DMBSPostgres tests here (and remove the option from the test struct because it's always on)
* Move all the other tests together with another test suite which already contains a wide variety of tests.

For the standard configuration, we want to have a very large test suite containing all of them, not have them spread out throughout smaller tests.

I hope that makes sense.